nixos/modules/services/desktops/gnome3/at-spi2-core.nix

```diff
@@ -18,6 +18,9 @@ with lib;
         description = ''
           Whether to enable at-spi2-core, a service for the Assistive Technologies
           available on the GNOME platform.
+
+          Enable this if you get the error or warning
+          <literal>The name org.a11y.Bus was not provided by any .service files</literal>.
```

Comment:
I don't think we should add these. We more need a mechanism to check if you're missing a certain dbus interface.

Reply:
We don’t have the mechanism, so this is the next best thing requiring the least effort.

Reply:
@jtojnar I believe a note like this used to be in the package description?
I'm still not sure how I feel about it *shrug.

Reply:
well, it would have saved me from 10 minutes of random internet search, that’s for sure